yt/frontends/parthenon/tests/test_outputs.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
```

Comment:
looks like this module was copied straight from the Athena++ frontend with no modification. It's a good idea to use it as a basis but we'll need to iterate on this